src/index.html

```diff
@@ -1,17 +1,115 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <header class="header">
+      <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+        <div class="personal__info">
+          <fieldset>
+            <legend>Personal information</legend>
+            <label for="surname">Surname:</label>
+            <input 
+            type="text" 
+            id="surname" 
+            name="surname" 
+            required 
+            autocomplete="off"
+            >
```

Comment:
fix code all code styling according to checklist
```suggestion
        <div class="personal__info">
          <fieldset>
            <legend>Personal information</legend>
            
            <label for="surname">Surname:</label>
            
            <input 
              type="text" 
              id="surname" 
              name="surname" 
              required 
              autocomplete="off"
            >
```